config/crd/bases/management.project-flotta.io_edgedevices.yaml

```diff
@@ -118,6 +118,20 @@ spec:
                         type: integer
                     type: object
                 type: object
+              metricsReceiver:
+                properties:
+                  ? "R\requestNumSamples"
+                  : description: request num samples
```

Comment:
If you add description, please describe what the property means/what it is for.

Reply:
It was added automatically by generate. not up to me

Reply:
It comes from here: https://github.com/project-flotta/flotta-operator/pull/148/files#diff-d751bc14a529b6b257da989fff2cafcd3797143688613e6fba8b8616ba27e46dR45

Reply:
now I understand. When moving to EdgeDevice CR I copied from the generated swagger file into this file.

Reply:
fixed